user/nix/files/nix.profile.d

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+if [ -z "$__NIX_SET_ENV_DONE" ]; then
+	. /etc/default/nix
+
+	# default paths
+	DATA_PATH="${XDG_DATA_DIRS-/usr/local/share:/usr/share}"
+	XDG_STATE_HOME="${XDG_STATE_HOME-$HOME/.local/state}"
+
+	# use nix command to parse settings so we don't have to
+	case `nix --extra-experimental-features nix-command config show use-xdg-base-directories` in
```

Comment:
This is just nit picking. Nix sometimes unexpectedly goes to the Internet to update substitutes. It doesn't hurt being extra cautious here.

```suggestion
	case "$(nix --offline --experimental-features nix-command config show use-xdg-base-directories)" in
```